.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
These .idea files should be added to .gitignore and removed from the repo :+1:

Reply:
Wrong checkin :) , thanks removed it